ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
@junr03 That's great to know! I infer you will make the change to the docker_push.sh script separately - please let me know if you want me to add the push in there and I can update the PR accordingly.

Regarding the unstripped binary - its size is around 150MB, which is still quite large, as opposed to the stripped one which is only ~8MB. In production environments, we'd rather not have debug enabled binaries unless debugging an issue (in which case we could deploy images with the unstripped binary).

Can we have CI push two images in such a case, one stripped and one debug?